### ISSUE TITLE
[SysTray icon request] Systray icon is needed for Steam

### ISSUE THREAD

**Issue by sergioad** [NitruxSA/plasma-next-icons#117]
Hello @UriHerrera and all the Nitrux team behind this awesome icon theme, A Breeze icon is needed for the Steam's tray icon, it is normally white so it does not fits with the general KDE / plasma / breeze guidelines, I think an icon for it could be awesome :smile:

**Comment by Steinhagen** [comment]
The default Steam tray icon under Breeze:
![steamicon](https://cloud.githubusercontent.com/assets/4029937/7943709/eebc4c0e-096e-11e5-9dee-ac8430d6b8a2.png)

**Comment by sergioad** [comment]
@Steinhagen yes, it does not fits on the breeze's white, this is why a breeze symbolic icon is needed

**Comment by UriHerrera** [comment]
Breeze doesn't use symbolic icons that said the icon will not be in the main theme it will be in a different branch.

**Comment by sergioad** [comment]
@UriHerrera I mean that a black version for breeze and a light version of the icon for Breeze dark

**Comment by sergioad** [comment]
I said symbolic because it is a symbol

**Comment by G-Ray** [comment]
+1

**Comment by nortexoid** [comment]
A matching icon would be very nice indeed.
